### DISCUSSION TITLE
dark mode based on system

### DISCUSSION THREAD

**Discussion by lilgue** [t1m0thyj/WinDynamicDesktop#315] [Ideas]
i'd like to see the wallpaper changing automatically with the system

**Comment by t1m0thyj** [comment]
Please explain what you mean by "with the system".

Windows does not support automatically switching between light and dark mode. You can install a 3rd party app like [Auto Dark Mode](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode) to do this, or WDD can also do it using [scripts](https://github.com/t1m0thyj/WDD-scripts/tree/master/stable#change-windows-app-theme).
